services/core-api/app/api/mines/region/models/region.py

```diff
@@ -14,12 +14,12 @@ class MineRegionCode(AuditMixin, Base):
     effective_date = db.Column(db.DateTime, nullable=False, server_default=FetchedValue())
     expiry_date = db.Column(db.DateTime)
 
-    regional_contacts = db.relationship('RegionalContact', lazy='select')
+    regional_contacts = db.relationship('EMLIContact', lazy='select')
```

Comment:
Can you add a TODO above this at the least to rename it and all its uses for the future at the very least

Reply:
RegionalContact not longer exist and it was replaced in all its uses by EMLIContact. Should I add a note on this regard?